web/src/beta/lib/core/StoryPanel/SelectableArea/index.tsx

```diff
@@ -49,15 +49,7 @@ const SelectableArea: React.FC<Props> = ({
   onClickAway,
   onRemove,
 }) => {
-  const {
-    isHovered,
-    showPadding,
-    panelSettings,
-    setShowPadding,
-    handleMouseOver,
-    handleMouseOut,
-    handleClickAway,
-  } = useHooks({
+  const { showPadding, panelSettings, setShowPadding, handleClickAway } = useHooks({
```

Comment:
You removed some values from the destructured hooks, but seem to not remove them from the hooks. If they aren't being used anymore, please update the hooks file too 🙏🏻